src/vivarium_census_prl_synth_pop/components/observers.py

```diff
@@ -69,6 +69,10 @@ def output_filename(self):
     def input_columns(self):
         pass
 
+    @property
+    def input_values(self):
+        return []
```

Comment:
If you define `INPUT_VALUES` as an empty list in the base class, you can have this method return `self.INPUT_VALUES` and then you don't need to define the property in the `WICObserver`